katana/katana.py

```diff
@@ -90,6 +93,207 @@ def readconfig():
     cfg = json.loads("".join(lines))
     return cfg
 
+@route('/searchkw', method='POST')
+def searchkw():
+    """
+    This method returns the ActionFile Path of the selected Driver.
+    """
+    value = parseString("".join(request.body))
+    indented_xml = "".join(value.toprettyxml(newl='\n'))
+    corrected_xml = remove_extra_newlines_char_xml(indented_xml)
+    corrected_xml = corrected_xml
+    with open('obj.txt', 'w') as f:
```

Comment:
relative paths won't work if user isn't launching katana at the current working directory

Reply:
I understand the issue @kaheichan . I will address it. Thank you.

Reply:
Working on this comment.

Reply:
Done @kaheichan